openapi_python_client/templates/model.pyi

```diff
@@ -105,27 +106,69 @@ class {{ model.reference.class_name }}:
             {{ construct(model.additional_properties, "prop_dict") | indent(12) }}
             additional_properties[prop_name] = {{ model.additional_properties.python_name }}
 
-        {{model.reference.module_name}}.additional_properties = additional_properties
+        {{model.reference.module_name}}._additional_properties = additional_properties
     {% else %}
-        {{model.reference.module_name}}.additional_properties = d
+        {{model.reference.module_name}}._additional_properties = d
```

Comment:
I'm surprised nothing else in `from_dict` had to change. 

I don't see an override for the constructor, so wouldn't you also have to prefix the kwargs to `cls(...)` above with `_` as well? Or is the constructor override coming up later?

Reply:
`attrs` automatically removes `_` prefixes for arguments to the constructor.

Reply:
TIL